src/i18n.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
 import i18n from "i18next";
 import { initReactI18next } from "react-i18next";
 // import Backend from 'i18next-http-backend';
```

Comment:
should be removed

Reply:
I added it because there was an error for importing common_en, common_ar and common_tr